classes/class-p4-master-site.php

```diff
@@ -85,7 +85,6 @@ protected function hooks() {
 		add_filter( 'get_twig', [ $this, 'add_to_twig' ] );
 		add_action( 'init', [ $this, 'register_taxonomies' ], 2 );
 		add_action( 'init', [ $this, 'register_oembed_provider' ] );
-		add_action( 'pre_get_posts', [ $this, 'add_search_options' ] );
```

Comment:
This only affects the main query, however our search doesn't use the main query (even though it's still executed) so there's no use in adding this action any more.